README.md

```diff
@@ -108,7 +108,7 @@ var rsa_key = {
 var cryptographer = new Jose.WebCryptographer();
 cryptographer.setContentSignAlgorithm("RS256");
 
-var signer = new JoseJWS.Signer(cryptographer);
+var signer = new Jose.JoseJWS.Signer(cryptographer);
 signer.addSigner(rsa_key).then(function() {
   signer
     .sign("hello world", null, {})
```

Comment:
Feel free to add your name to the list of contributors (below in this file). This is a major contribution, I appreciate the time & knowledge you are sharing with us!